wcivf/apps/elections/templates/elections/postcode_view.html

```diff
@@ -62,19 +62,22 @@
 
             {% if postelections %}
                 {% include "feedback/feedback_form.html" %}
+                <div id="report_form_container" style="display: none;">
```

Comment:
I don't think we should inline the style at the point we include the template. If it's meant to be hidden by default, then that should be left up to the template we're including. Can you change this to just be the include line, and move the wrapping div to the included template?